build.gradle

```diff
@@ -150,6 +153,11 @@ dependencies {
     modImplementation("dev.ftb.mods:ftb-library-forge:${project.ftb_library_version}") { transitive = false }
     modImplementation("dev.ftb.mods:ftb-teams-forge:${project.ftb_teams_version}") { transitive = false }
 
+    modRuntimeOnly("com.jozufozu.flywheel:flywheel-forge-1.20.1:0.6.10-7")
+    modRuntimeOnly("curse.maven:create-328085:4835191") { transitive = false }
+    modImplementation("blank:Steam_Rails-1.5.3+forge-mc1.20.1") // fixed Accessor
```

Comment:
do they not have a maven?

Reply:
they do. `https://maven.ithundxr.dev/releases`

Reply:
Added, still crashing but still added